inst/bench/README.Rmd

```diff
@@ -19,6 +19,8 @@ here::i_am("inst/bench/README.Rmd")
 library(dplyr)
 library(ggplot2)
 library(readr)
+library(dplyr)
```

Comment:
already included three lines above?

Reply:
I will fix that but this will not be part of the submitted package.